docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '2.4'
+
+services:
+  rabbitmq:
+    image: rabbitmq:3-management-alpine
+    container_name: rabbitmq-ef
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    healthcheck:
+      test: [ "CMD", "nc", "-z", "localhost", "5672" ]
+      interval: 1s
+      timeout: 5s
+      retries: 60
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.5
+    container_name: elasticsearch-ef
+    environment:
+      - discovery.type=single-node
+      - ES_JAVA_OPTS=-Xms512m -Xmx512m
+    ports:
+      - "9200:9200"
+    mem_limit: 1g
```

Comment:
I tried various configuration but the only one working is 4g with `ES_JAVA_OPTS=-Xms1g -Xmx1g`